instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/package.py

```diff
@@ -14,3 +14,4 @@
 
 
 _instruments = ("botocore ~= 1.0",)
+_aiobotocore_instruments = ("aiobotocore ~= 2.0",)
```

Comment:
This should looks something like the following so that opentelemetry-bootstrap will work with that.

```
_instruments_botocore = "botocore ~= 1.0"
_instruments_aiobotocore = "aiobotocore ~= 2.0"

_instruments = ()
_instruments_any = (_instruments_botocore, _instruments_aiobotocore)
```